survey-sdk/src/main/java/co/nimblehq/survey/sdk/model/SurveyModel.kt

```diff
@@ -0,0 +1,5 @@
+package co.nimblehq.survey.sdk.model
+
+data class SurveyModel(
+    val surveyId: String, val surveyTitle: String, val surveyDescription: String
```

Comment:
I think you will need to run code format here, oftentimes it will break into lines if I'm not wrong.
If you use AS to commit + push, check the box for the vode to be auto-formatted before committing

Reply:
@sleepylee I setup for AS and do manually as well.
But the result is still same.

Reply:
<img width="973" alt="Screen Shot 2021-09-21 at 10 43 06 PM" src="https://user-images.githubusercontent.com/88996251/134203066-03963d3e-f371-4ffe-a362-7679089e25e2.png">